specs/src/specs/data_structures.md

```diff
@@ -112,11 +111,7 @@ Abstraction over transaction fees.
 
 ### Address
 
-Address is a [type alias](#type-aliases).
-
-Addresses are the [hash](#hashing) [digest](#hashdigest) of the [public key](https://docs.cosmos.network/v0.46/basics/accounts.html#public-keys).
-
-Addresses have a length of 32 bytes.
+Celestia supports [secp256k1](https://en.bitcoin.it/wiki/Secp256k1) keys where [addresses](https://docs.cosmos.network/v0.46/basics/accounts.html#addresses) are 20 bytes in length. Addresses are prefixed with the [Bech32](https://en.bitcoin.it/wiki/Bech32) prefix `celestia`. For example, a valid address is `celestia1kj39jkzqlr073t42am9d8pd40tgudc3e2kj9yf`.
```

Comment:
The HRP of the bech32 encoding (and indeed the encoding itself) is orthogonal to the actual address value. I think the specs should have a section for the value, and a section for the human-readable encoding used by frontends.

Reply:
I can extract a new section for the human-readable encoding but it seems like it doesn't belong on this page (data_structures.md). Proposal to extract a new page for `Public-Key Cryptography` which contains the section from this page along with a new section for human-readable Address encoding.

Reply:
Done in https://github.com/celestiaorg/celestia-app/pull/1553/commits/cb52073e9cd3f9f69ef432d9d408db97c7767046